TeamFourtressGame/core/src/com/fourtress/views/MenuScreen.java

```diff
@@ -97,7 +97,6 @@ public void hide() {
 
 	@Override
 	public void dispose() {
-		stage.dispose();

```

Comment:
Is this right? Apart from this the rest of the changes look good to me! :D

Reply:
@kieranswhite  : Yes, as discussed - if we dispose the menu screen and the player runs out of time via the timer we display the finish screen which gives the user two options: to either go back to the main menu or retry the game. I have removed the above statement because the screen will no longer "exist" and you will just get a black screen if the user chooses to return to the main menu as screen was disposed on the initial transition from the main menu to the game screen... if that makes sense?